src/components/AprInfo/AprInfo.tsx

```diff
@@ -54,6 +55,7 @@ export function AprInfo({
   const isIncentiveActive = !!incentivesData && incentivesData?.rewardsPerMarket[tokenAddress] !== undefined;
   const isLidoApr = lidoApr !== undefined && lidoApr > 0n;
   const isEthenaSatsIncentive = isEthenaSatsIncentivizedMarket(tokenAddress);
+  const isTbtcIncentive = isTbtcIncentivizedMarket(tokenAddress);
```

Comment:
just noticed tokenAddress is actually a marketAddress, let's please rename this prop to avoid confusion

Reply:
indeed, done

Reply:
indeed, done